contracts/FT-voting-simple/RIFGovernorFT.sol

```diff
@@ -37,7 +43,9 @@ contract RIFGovernorFT is
             descriptionHash
         );
         // call function on the target smart contract after the proposal execution
-        proposalTarget.onProposalExecution(proposalId);
+        if (address(proposalTarget) != address(0)) {
```

Comment:
Is there a situation where we want to allow a `null` proposal target?
- If yes, add a comment here to state that, and why
- If no, replace the `if` with a `require`.